src/__tests__/__snapshots__/typescript.js.snap

```diff
@@ -38,13 +38,7 @@ test/should-fail.test.tsx(100,24): error TS2551: Property 'colors' does not exis
 test/should-fail.test.tsx(111,3): error TS2344: Type 'PropsWithoutTheme' does not satisfy the constraint '{ theme: any; }'.
   Property 'theme' is missing in type 'PropsWithoutTheme'.
 test/should-fail.test.tsx(119,3): error TS2345: Argument of type 'StatelessComponent<object>' is not assignable to parameter of type '\\"tspan\\"'.
-test/should-fail.test.tsx(134,3): error TS2345: Argument of type '(props: { theme: any; } & ExampleComponentProps & object) => { display: \\"none\\" | \\"hidden\\"; }' is not assignable to parameter of type 'StyleArgument<CSSProperties, { theme: any; } & ExampleComponentProps & object>'.
```

Comment:
Those are in should-fail tests, so I think you should remove relevant tests too. I mean, line 134 in test/should-fail.test.tsx.

Reply:
These tests will be valid again once https://github.com/frenic/csstype/issues/8 is merged. Would it make sense to just add comments to the effect that the lines are only temporarily not being flagged with errors, pending that PR?

Reply:
@pelotom Oh, sorry. I though https://github.com/frenic/csstype/issues/8#issuecomment-364845647 says that '`string` is valid', but it does not... Sorry for giving confusion.